staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/initializer.go

```diff
@@ -18,13 +18,15 @@ package validatingadmissionpolicy
 
 import (
 	"context"
+
 	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
 )
 
 type CELPolicyEvaluator interface {
 	admission.InitializationValidator
 
-	Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error
+	Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces, authorizer authorizer.Authorizer) error
```

Comment:
same question about whether the authorizer is an attribute of the request (a param here) or an attribute of the validator instance (passed to construction). I would have expected the latter.

Reply:
Thanks, fixed!